Documentation/commands.md

```diff
@@ -156,4 +205,22 @@ Work in progress. Please contribute!
 
 ### rkt gc
 
-Work in progress. Please contribute!
\ No newline at end of file
+Rocket has a built-in garbage collection command that is designed to be run periodically from a timer or cron job. Containers older than the grace period are marked and cleaned up during a subsequent garbage collection pass. [Read more about the container lifecycle][gc-docs].
+
+[gc-docs]: container-lifecycle.md#garbage-collection
+
+```
+$ rkt gc --grace-period=30m0s
+Moving container "21b1cb32-c156-4d26-82ae-eda1ab60f595" to garbage
+Moving container "5dd42e9c-7413-49a9-9113-c2a8327d08ab" to garbage
+Moving container "f07a4070-79a9-4db0-ae65-a090c9c393a3" to garbage
+```
+
+On the next pass, the containers are removed:
```

Comment:
Actually it's the other way around, the grace period is considered here, not in the previous step.